### DISCUSSION TITLE
PAL NES power issue?

### DISCUSSION THREAD

**Discussion by hylobates-agilis** [darthcloud/BlueRetro#1010] [Q&A]
Hi everyone,

I’m trying to get a Blueretro working on a PAL NES with a [NESRGB kit](https://www.etim.net.au/nesrgb/) and it does not work properly when being powered by the console alone: it powers on but does not get into blue pulsing light for inquiry mode, controller does not connect, intput are completely erratic, blue light powers on every 10 sec or so but that’s it.

When using micro-USB to power it, it seems fine. Reading the PIN1 to PIN5 voltage on the controller port on the NES, I get a 4.82V which seems a bit low.

Am I right assuming that I have power issues on the controller ports? Is there a way to fix this (like changing the power supply or the voltage regulator for example)?

Thanks in advance.

**Comment by hylobates-agilis** [comment]
I asked a few friends to test their own NES, and they all show 4.8V on the controller port… I really don’t know what to do with this information though.

**Comment by mzryz** [comment]
I'm having the same problem with my NES too. It's also NESRGB modded. My blueretro core has been updated to the latest universal firmware, it works on my Megadrive, Saturn but not my NES. As you say, I don't get blue pulsing, I get a blue flash every few seconds and I dont know what that means yet. Sad to see that you haven't had any support on this issue since May. I hope we can get a fix somehow.

**Reply by hylobates-agilis** [reply]
Asked [on lemmy](https://jlai.lu/post/7427825) a few month ago, it’s most probably bad quality cables.

**Reply by mzryz** [reply]
Ok I will check the cables with my multimeter and see. I guess you haven't fixed yours yet?

**Reply by hylobates-agilis** [reply]
No, I haven’t. I’m currently using a micro-USB external power supply to avoid the issue. I’m planning on building an adapter myself to see if I can completely circumvent the issue.

**Comment by mzryz** [comment]
@darthcloud would you have any ideas we can try? Thank you so much

**Comment by darthcloud** [comment]
I don't think the issue is the voltage value. It's probably that coupled with the RGB mod the NES might not provide enough current.

You best bet is to continue powering externaly.
